src/Validator/Formatter.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Overblog\GraphQLBundle\Validator;
+
+use Overblog\GraphQLBundle\Event\ErrorFormattingEvent;
+use Overblog\GraphQLBundle\Validator\Exception\ArgumentsValidationException;
+
+class Formatter
+{
+    public function onErrorFormatting(ErrorFormattingEvent $event): void
+    {
+        $error = $event->getError()->getPrevious();
+
+        if ($error instanceof ArgumentsValidationException) {
+            $state = [];
+            $code = [];
+
+            $violations = $error->getViolations();
+            foreach ($violations as $violation) {
+                $state[$violation->getPropertyPath()][] = $violation->getMessage();
+                $code[$violation->getPropertyPath()][] = $violation->getCode();
+            }
+
+            $formattedError = $event->getFormattedError();
+            $formattedError->offsetSet('state', $state);
```

Comment:
error should contain only `message`, `locations`, `path` and `extensions` as root entry  all additional entry should be add under `extensions` (see [the specs](https://graphql.github.io/graphql-spec/June2018/#sec-Errors))

Reply:
Will be refactored according to the specification.

Reply:
Done